themes/base/tooltip.css

```diff
@@ -13,8 +13,8 @@
 	position: absolute;
 	z-index: 9999;
 	max-width: 300px;
-	-webkit-box-shadow: 0 0 5px #aaa;
-	box-shadow: 0 0 5px #aaa;
+	-webkit-box-shadow: 0 0 5px #ddd;
+	box-shadow: 0 0 5px #ddd;
```

Comment:
@scottgonzalez - I didn't because @jzaefferer already created PR #1420. I think that PR, together with https://github.com/jquery/download.jqueryui.com/pull/248, covers everything that needs to be done. Or did I miss something?